test/rules/accessibility/tooltip-accessibility.spec.js

```diff
@@ -0,0 +1,112 @@
+var expect = require("expect.js");
+
+var HTMLHint = require("../../../index").HTMLHint;
+
+var ruldId = 'tooltip-accessibility',
+    ruleOptions = {};
+
+ruleOptions[ruldId] = true;
+var noDescribedElementError = 'tooltip {0} expects element to describe';
+var tooltip = `<span id="description" class='tooltip-help'></span>`;
+
+var formattedMessage = function (param) {
+    return noDescribedElementError.replace('{0}', param);
+};
+
+describe('Rules: ' + ruldId, function () {
+
+    it('span with class "tooltip-help" without id should result in an error', function () {
+        var code = `<div>
+                    <span class='tooltip-help'></span>
+                    <input data-bind='addDescription:"input description",lock:true' />
+                    </div>`;
+        var messages = HTMLHint.verify(code, ruleOptions);
+        expect(messages.length).to.be(1);
+        expect(messages[0].rule.id).to.be(ruldId);
+        expect(messages[0].message).to.equal('span with class tooltip-help must have an id');
+        // expect(messages[0].line).to.be(2);
+        // expect(messages[0].col).to.be(21);
+    });
+
+    it('tooltip with no described element should result in an error', function () {
+        var code = `<div>
+                   ${tooltip}
+                    </div>`;
+        var messages = HTMLHint.verify(code, ruleOptions);
+        expect(messages.length).to.be(1);
+        expect(messages[0].rule.id).to.be(ruldId);
+        expect(messages[0].message).to.equal(formattedMessage(tooltip));
+        // expect(messages[0].line).to.be(3);
```

Comment:
uncomment and add the correct line/column numbers here and in the following test cases